README.md

```diff
@@ -41,6 +41,9 @@ databaseds.workspace   is running on: 172.18.0.3:10000
 mtango.workspace       is running on: 172.18.0.5:8080
 ```
 
+Remember, `in docker-compose.yml`, taurus section, you have to set DISPLAY 
+to `your_ip_addres:0`
+
```

Comment:
Can you please write another sentence that one has to have an X server (such as Xming) running on his machine?